app/controllers/invitations_controller.rb

```diff
@@ -30,6 +30,7 @@ def create
     @invitation.inviter = current_user
     @invitation.token = Digest::SHA1.hexdigest([@invitation.sender_id, Time.now, rand].join)
     @team = Team.find(@invitation.sender_id)
+    @invitation.email = @invitation.email.downcase!
```

Comment:
Consider using a before_filter in the model to handle this situation: http://guides.rubyonrails.org/action_controller_overview.html#filters